kitsune/sumo/email_utils.py

```diff
@@ -22,11 +22,13 @@ def send_messages(messages):
     if not messages:
         return
 
-    conn = mail.get_connection(fail_silently=True)
-    conn.open()
-
-    for msg in messages:
-        conn.send_messages([msg])
+    try:
+        conn = mail.get_connection()
+        conn.open()
+        for msg in messages:
+            conn.send_messages([msg])
+    finally:
+        conn.close()
```

Comment:
Nit: I'm pretty sure you don't need the `conn.close()` because it will close automatically when the function ends and things get cleaned up.

Other than that, I think this looks good. Depending on what happens next, we might want to create message retrying. If we do that, then we're going to want to keep track of which messages succeeded and which failed. We might find that `.send_messages()` should be in a try/except block so that if we hit a failure, we skip just one message rather than that message and the rest in the list. I don't know offhand, though. I think this is good for now, but we might want to file a followup bug to hone it next week depending on what happens.